eras/shelley/test-suite/src/Test/Cardano/Ledger/Shelley/Generator/Core.hs

```diff
@@ -725,50 +700,13 @@ genesisAccountState =
 -- | Creates the UTxO for a new ledger with the specified
 -- genesis TxId and transaction outputs.
 genesisCoins ::
+  CC.Crypto (Crypto era) =>
   Ledger.TxId (Crypto era) ->
   [Core.TxOut era] ->
   UTxO era
 genesisCoins genesisTxId outs =
   UTxO $
-    Map.fromList [(TxIn genesisTxId idx, out) | (idx, out) <- zip [0 ..] outs]
-
--- | Apply a transaction body as a state transition function on the ledger state.
-applyTxBody ::
-  forall era.
-  ( Era era,
-    Show (Core.TxOut era),
-    HasField "inputs" (Core.TxBody era) (Set (TxIn (Crypto era))),
-    HasField "certs" (Core.TxBody era) (StrictSeq (DCert (Crypto era))),
-    HasField
-      "wdrls"
-      (Core.TxBody era)
-      (Wdrl (Crypto era))
-  ) =>
-  LedgerState era ->
-  PParams era ->
-  Core.TxBody era ->
-  LedgerState era
-applyTxBody ls pp tx =
-  ls
-    { _utxoState =
-        us
-          { _utxo = eval (txins @era tx ⋪ (_utxo us) ∪ txouts tx),
-            _deposited = depositPoolChange ls pp tx,
-            _fees = (getField @"txfee" tx) <> (_fees . _utxoState $ ls)
-          },
-      _delegationState =
-        dels
-          { _dstate = dst {_rewards = newAccounts}
-          }
-    }
-  where
-    dels = _delegationState ls
-    dst = _dstate dels
-    us = _utxoState ls
-    newAccounts =
-      reapRewards
-        ((_rewards . _dstate . _delegationState) ls)
-        (Map.mapKeys getRwdCred . unWdrl $ getField @"wdrls" tx)
+    SplitMap.fromList [(TxIn genesisTxId idx, out) | (idx, out) <- zip [0 ..] outs]
 
```

Comment:
I don't understand what is happening here.  What did we replace the lines 733-771 with?

Reply:
`applyTxBody` is dead code. It is not anywhere in the codebase, thus I removed it

Reply:
great catch, thank you @lehins !